api/handlers_metrics.go

```diff
@@ -378,7 +313,16 @@ func (s *server) GetRDSMetricsURLHandler(w http.ResponseWriter, r *http.Request)
 
 	cwMetrics := []cloudwatch.Metric{}
 	for _, m := range metrics {
-		cwMetrics = append(cwMetrics, cloudwatch.Metric{"AWS/RDS", m, "DBInstanceIdentifier", instanceId})
+		switch queryType {
+		case "instance":
+			cwMetrics = append(cwMetrics, cloudwatch.Metric{"AWS/RDS", m, "DBInstanceIdentifier", instanceId})
+		case "cluster":
+			cwMetrics = append(cwMetrics, cloudwatch.Metric{"AWS/RDS", m, "DBClusterIdentifier", instanceId})
+		default:
+			msg := fmt.Sprintf("invalid metric requested: %s", m)
```

Comment:
This should say `invalid type requested`

Reply:
Good catch